cwod_site/local_settings_docker.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql', 		 # Add 'sqlite3', postgresql_psycopg2', 'postgresql', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'capitolwords',                      # Or path to database file if using sqlite3.
+        'USER': 'capitolwords',                      # Not used with sqlite3.
+        'PASSWORD': 'capitolwords',                  # Not used with sqlite3.
+        'HOST': 'db',                      	 # Set to empty string for localhost. Not used with sqlite3.
+        'PORT': '3306',                      	     # Set to empty string for default. Not used with sqlite3.
+    }
+}
+
+TEMPLATE_DIRS = (
+	os.path.dirname(os.path.realpath(__file__)) + '/templates/'
+)
+
+NYT_API_KEY = ''
```

Comment:
Update to `PROPUBLICA_API_KEY`.

Is there a reason we need this file versus the `local_settings.example.py`? Would be nice not to have to maintain both files and this looks identical.